games-action/minetest_game/minetest_game-0.4.15.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit vcs-snapshot
+
+DESCRIPTION="The main game for the Minetest game engine"
+HOMEPAGE="https://github.com/minetest/minetest_game"
+SRC_URI="https://github.com/minetest/${PN}/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="GPL-2 CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=games-action/minetest-${PV}"
+
+src_install() {
+	insinto /usr/share/minetest/games/${PN}
```

Comment:
nah, insinto is a Gentoo command, and is prefix aware, not need to add `${EPREFIX}`